### DISCUSSION TITLE
We have released a new version based on LSP4IJ

### DISCUSSION THREAD

**Discussion by CppCXY** [redhat-developer/lsp4ij#387] [Show and tell]
We have released a new version and look forward to more people learning about LSP4IJ!
see: https://github.com/LuaLS/lua-language-server/issues/2728

**Comment by angelozerr** [comment]
It is a great news! Thanks @CppCXY for your feedback. 

Hope people will like LSP4IJ and thanks for help us to promote LSP4IJ.

**Comment by fbricon** [comment]
@CppCXY congrats! Now I guess you can update https://github.com/CppCXY/Intellij-SumnekoLua?tab=readme-ov-file#introduce
